app/lib/actions/pulp/orchestration/repository/sync.rb

```diff
@@ -0,0 +1,20 @@
+module Actions
+  module Pulp
+    module Orchestration
+      module Repository
+        class Sync < Pulp::Abstract
+          def plan(repository, smart_proxy, options)
+            sequence do
+              input[:output] = plan_action(Actions::Pulp::Repository::Sync, options).output
+              plan_self(input)
```

Comment:
same comment here, can we just pass things to plan_self() explicitly, instead of stuffing it in the input and then passing it?